src/game/C4Application.cpp

```diff
@@ -450,6 +450,19 @@ void C4Application::ParseCommandLine(int argc, char * argv[])
 				Game.DirectJoinAddress[0] = 0;
 				continue;
 			}
+			// Special case: start the mod dialog and initiate installation of a mod.
+			if (SEqualNoCase(Game.DirectJoinAddress, "installmod", 10))
+			{
+				// IDs are at least a few characters long.
+				const char *id = Game.DirectJoinAddress + 10;
+				if (SLen(id) > 10)
+				{
+					++id; // Remove slash.
+					C4Startup::SetStartScreen("mods", id);
+				}
+				Game.DirectJoinAddress[0] = 0;
+				continue;
+			}
```

Comment:
The 10 appears multiple times here - if they all have the same meaning it would be good to use a constant instead of 10. No idea what this magic number is good for, could you add a comment? Looks like some kind of prefix for determining the ID

Reply:
Yes, true. I can make that a bit more readable. The 10 ist just strlen("installmod") :)